alerts/google-gke/uptime-checks-workload.v1.json

```diff
@@ -0,0 +1,37 @@
+{
+  "displayName": "${CLUSTER_NAME}/${WORKLOAD_NAME} GKE Load Balancer Check uptime failure"
+  "documentation": {},
+  "userLabels": {
+    "workload_name": "${WORKLOAD_NAME}",
+    "cluster": "${CLUSTER_NAME}",
+    "uptime_check_id": "${UPTIME_CHECK_ID}",
+  },
+  "conditions": [
```

Comment:
Missing conditions.displayName -> "Failure of ${alertPolicy.displayName}"

Reply:
not sure how we'll populate the alert policy display name, i think it would make sense to put the uptime check name here?